src/components/plugin-wrapper/plugin-wrapper.tsx

```diff
@@ -29,11 +29,11 @@ export const PluginWrapper: FC<PluginWrapperProps> = ({
 }) => {
     const [hasAnalyticsData, setHasAnalyticsData] = useState(false)
 
-    const onResponseReceived = useCallback(
-        (args) => {
+    const onResponseReceived = useCallback<OnAnalyticsResponseReceivedCb>(
+        (items, dimensions, headers) => {
             setHasAnalyticsData(true)
 
-            onResponsesReceivedCb?.(args)
+            onResponsesReceivedCb?.(items, dimensions, headers)
```

Comment:
As mentioned before, I think this should be more "generic" and accept different parameters because `headers` only applies to LL (to be confirmed).

Reply:
Yes true, as discussed, we'll see how we deal with this once we have the PT in place.